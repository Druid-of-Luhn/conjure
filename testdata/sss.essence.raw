Spec
  { language = "Essence"
  , version = [ 2
              , 0
              ]
  , topLevelBindings = [ ( Find
                         , "x"
                         , DomainSet
                             { size = Nothing
                             , minSize = Nothing
                             , maxSize = Nothing
                             , attrDontCare = False
                             , element = DomainSet
                                           { size = Nothing
                                           , minSize = Nothing
                                           , maxSize = Nothing
                                           , attrDontCare = False
                                           , element = DomainSet
                                                         { size = Nothing
                                                         , minSize = Nothing
                                                         , maxSize = Nothing
                                                         , attrDontCare = False
                                                         , element = DomainIntegerFromTo (
                                                                       Just ( ValueInteger 0 ) ) (
                                                                       Just ( ValueInteger 9 ) )
                                                         , representation = Nothing
                                                         }
                                           , representation = Nothing
                                           }
                             , representation = Nothing
                             }
                         )
                       ]
  , topLevelWheres = []
  , objective = Nothing
  , constraints = [ GenericNode
                      Eq
                      [ GenericNode
                          Card
                          [ Identifier "x"
                          ]
                      , ValueInteger 2
                      ]
                  ]
  , metadata = []
  }