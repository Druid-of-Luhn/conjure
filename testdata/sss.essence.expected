language Essence 2.0

find x : set of set of set of int(0..9)

such that
    card(x) = 2
